.github/workflow-gen/Program.cs

```diff
@@ -39,7 +39,8 @@ void GenerateCiWorkflow(Product product)
     {
         $".github/workflows/{product.Name}-**",
         $"{product.Name}/**",
-        "Directory.Packages.props"
+        "Directory.Packages.props",
+        ".editorconfig"
```

Comment:
Nitpick, but should we put this above `$".github/workflows/{product.Name}-**"` to maintain sorting?

If so, I would also move `$"{product.Name}/**"` to the bottom, so that the statically named entries are sorted, followed by the dynamically named entry.

Reply:
Or `$"{product.Name}/**" could be at the top.

Reply:
Sorting is aligned with filesystem.